cmd/autoowners/main.go

```diff
@@ -133,40 +136,79 @@ func (orgRepo *orgRepo) getOwnersHTTP() (err error) {
 		data, err := gc.GetFile(orgRepo.Organization, orgRepo.Repository, filename, "HEAD")
 		if err != nil {
 			if _, nf := err.(*github.FileNotFound); nf {
+				logrus.WithField("orgRepo", fmt.Sprintf("%s/%s", orgRepo.Organization, orgRepo.Repository)).
+					WithField("filename", filename).
+					Debug("Not found file in the upstream repo")
+				orgRepo.OwnersFileExists = false
 				continue
 			} else {
 				return err
 			}
 		}
 
-		var target interface{}
+		orgRepo.OwnersFileExists = true
 		switch filename {
 		case "OWNERS":
-			target = &orgRepo.Owners
+			var target interface{}
+			target = &orgRepo.SimpleConfig
+			err = yaml.Unmarshal(data, target)
+			if err != nil {
+				logrus.WithField("data", string(data)).WithField("filename", filename).
+					WithField("orgRepo.Organization", orgRepo.Organization).
+					WithField("orgRepo.Repository", orgRepo.Repository).
+					WithError(err).Error("Unable to parse data for SimpleConfig.")
+				return err
+			}
+			if orgRepo.SimpleConfig.Empty() {
+				target = &orgRepo.FullConfig
+				err = yaml.Unmarshal(data, target)
+				if err != nil {
+					logrus.WithField("data", string(data)).WithField("filename", filename).
+						WithField("orgRepo.Organization", orgRepo.Organization).
+						WithField("orgRepo.Repository", orgRepo.Repository).
+						WithError(err).Error("Unable to parse data for FullConfig.")
+					return err
+				}
+				if len(orgRepo.FullConfig.Filters) == 0 {
+					return fmt.Errorf("invalid OWNERS file: %s", string(data))
```

Comment:
Why?

Reply:
Removed.

If it is empty simpleConfig and no filters in fullConfig ... Is it a valid `OWNERS` file? Maybe it is ... for `dirOptions`.
```
			if httpResult.simpleConfig.Empty() {
				...
				if len(httpResult.fullConfig.Filters) == 0 {
```